api/utils/health_utils.py

```
@@ -121,6 +130,105 @@ def get_oceanbase_status():
+        conn = _get_gaussdb_connection()
+        health_info = _mask_gaussdb_secret(conn.health())
+        performance_metrics = _mask_gaussdb_secret(conn.get_performance_metrics())
+        status = "alive" if health_info.get("status") == "healthy" else "timeout"
+        return {
+            "status": status,
+            "message": {
+                "health": health_info,
+                "performance": performance_metrics,
+            },
+        }
+    except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)